### ISSUE TITLE
请问model converter是不是还不支持caffe DepthwiseConvolution的转换？

### ISSUE THREAD

**Issue by WeixiangXu** [XiaoMi/mace#323]
我在/mace/ops中看到了depthwise_conv2d.cc的实现，但是在caffe_converter.py中没有看到这一类型的转换：

>     def __init__(self, option, src_model_file, src_weight_file):
        self._op_converters = {
            'Input': self.convert_nop,
            'Convolution': self.convert_conv2d,
            'Deconvolution': self.convert_deconv2d,
            'Eltwise': self.convert_elementwise,
            'Add': self.convert_add,
            'ReLU': self.convert_activation,
            'TanH': self.convert_activation,
            'Sigmoid': self.convert_activation,
            'PReLU': self.convert_activation,
            'Pooling': self.convert_pooling,
            'Concat': self.convert_concat,
            'Slice': self.convert_slice,
            'Softmax': self.convert_softmax,
            'InnerProduct': self.convert_fully_connected,
            'BatchNorm': self.convert_folded_batchnorm,
            'Crop': self.convert_crop,
            'Scale': self.convert_scale,
            'ShuffleChannel': self.convert_channel_shuffle,
            'Permute': self.convert_permute,
            'Flatten': self.convert_flatten,
            'PriorBox': self.convert_prior_box,
            'Reshape': self.convert_reshape,
        }

请问是不是暂不支持depthwiseConv的转换？

**Comment by lydoc** [comment]
支持，可以参考https://github.com/XiaoMi/mace/blob/a74002cb411c39098692b1ebf54c5ab0ac339304/mace/python/tools/converter_tool/caffe_converter.py#L378
